cabra/src/pages/components/RelatedTask.tsx

```diff
@@ -49,7 +48,12 @@ export default function RelatedTask({
   };
 
   return (
-    <Container>
+    <div
+      onClick={() => {
+        if (onClick != null) onClick(task.id);
+      }}
```

Comment:
```suggestion
      onClick={() => onClick?.(task.id)}
```
Looks better, doesn't it? 😄